core/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramParser.java

```diff
@@ -30,15 +34,10 @@
 /**
  *
  */
-public class DateHistogramParser extends HistogramParser {
```

Comment:
Can we fix this broken comment? Maybe we need something to fail the build, for files that contain this:

```
/**
 *
 */
```

What causes this? Is it Intellij?